### ISSUE TITLE
Install error with no clear issue in log

### ISSUE THREAD

**Issue by dethmourne** [rockerbacon/modorganizer2-linux-installer#464]
I'm getting "operation canceled, please see terminal for details" but this is my term:

INFO: all dependencies met
INFO: selected game 'skyrimspecialedition'
INFO: found Steam in '/home/loktai/.local/share/Steam'
INFO: steam not found in '/home/loktai/.var/app/com.valvesoftware.Steam/.local/share/Steam'
INFO: game found in '/home/loktai/.local/share/Steam'
INFO: selected install directory '/home/loktai/Games/mod-organizer-2-skyrimspecialedition'
INFO: using wget backend
INFO: fetching remote resource at 'https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u312-b07/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip'
INFO: downloading 'https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u312-b07/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip' to '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip'
INFO: extracting '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip' to '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07'
INFO: using wget backend
INFO: fetching remote resource at 'https://github.com/ModOrganizer2/modorganizer/releases/download/v2.4.4/Mod.Organizer-2.4.4.7z'
INFO: downloading 'https://github.com/ModOrganizer2/modorganizer/releases/download/v2.4.4/Mod.Organizer-2.4.4.7z' to '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4.7z'
INFO: extracting '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4.7z' to '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4'
INFO: using wget backend
INFO: fetching remote resource at 'https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks'
INFO: downloading 'https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks' to '/tmp/mo2-linux-installer-downloads-cache/winetricks'
INFO: using wget backend
INFO: fetching remote resource at 'https://skse.silverlock.org/beta/skse64_2_01_05.7z'
INFO: downloading 'https://skse.silverlock.org/beta/skse64_2_01_05.7z' to '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05.7z'
INFO: extracting '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05.7z' to '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05'
INFO: removing existing JDK installation in '/home/loktai/.local/share/Steam/steamapps/compatdata/489830/pfx/drive_c/java'
INFO: installing JDK in '/home/loktai/.local/share/Steam/steamapps/compatdata/489830/pfx/drive_c/java'
INFO: installing Mod Organizer 2 in '/home/loktai/Games/mod-organizer-2-skyrimspecialedition/modorganizer2'
INFO: installing script extender in '/home/loktai/.local/share/Steam/steamapps/common/Skyrim Special Edition'
INFO: copying '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05/skse64_2_01_05/Data' into '/home/loktai/.local/share/Steam/steamapps/common/Skyrim Special Edition'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05.7z'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/skyrimspecialedition_skse64_2_01_05'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4.7z'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/Mod.Organizer-2.4.4'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07.zip'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/OpenJDK8U-jre_x64_windows_hotspot_8u312b07'
INFO: removing '/tmp/mo2-linux-installer-downloads-cache/winetricks'

I don't see anything in that to clarify what could possibly be going wrong - it doesn't say the copy failed or anything, etc.

**Comment by EarlOfMagadan** [comment]
Distrohopped and reinstalled mo2, got this too

**Comment by tdtooke** [comment]
In the past I successfully used this, I rotated out Skyrim on games I play, uninstalled it completely along with this and just now today this happened.  Exact same output as in the first post.

**Comment by tdtooke** [comment]
This merits a new entry.  I cleaned house.  Uninstalled Skyrim and this time made sure to delete any and all folders/files that had anything to do with that.  This time it installed no problem.  I think the problem here is trying to install this thing twice without a proper uninstall, so to speak.

**Comment by mkalte666** [comment]
I can reliably reproduce this. 

  * Install mo2 
  * Purge it, **except forgetting to delete the skse artifacts in Data/Scripts** 
  * Try to install it again 
 
the copy command is not allowed to overwrite i suppose, and fails.

**Comment by SegoItCh** [comment]
Same problem

**Comment by dethmourne** [comment]
> I can reliably reproduce this.
> 
>     * Install mo2
> 
>     * Purge it, **except forgetting to delete the skse artifacts in Data/Scripts**
> 
>     * Try to install it again
> 
> 
> the copy command is not allowed to overwrite i suppose, and fails.

This ended up being the issue for me, after a Lutris-based attempt failed I had made the mistake of assuming it would clear everything out.

**Comment by dethmourne** [comment]
So for future readers who find this issue report and want to fix it --

1) Try uninstalling Skyrim and then manually delete anything left in the Skyrim folder afterwards  - Steam will not remove the subfolders used for SKSE scripts, for example.
2) Update protontricks - at least as of this writing, it appears that at least Ubuntu is on a bad/old version that causes issues.

After doing one or both of those steps as needed, try this all again. Ensure you are installing MO2 to an empty folder (you may need to delete/clear out any prior attempt's folder from ~/Games or wherever your failed install attempted to place it).

**Comment by SegoItCh** [comment]
@dethmourne problem still here, why close?

**Comment by dethmourne** [comment]
@SegoItCh The issue is (at least, for my report): "I didn't follow the instructions fully, including ensuring I had a clean install" - that's not something that rockerbacon's code needs to fix.

**Comment by UnstoppableMango** [comment]
I ran into this, as did another user who (as of typing) has an open issue in this repo. Maybe enterprise dev has gotten to my head, but although the issue has been resolved I think there is still some problem that could be fixed in code/scripts to help new and/or non-technical users from getting tripped up on such a simple thing as a file copy operation.

A README update would probably suffice, or as a more robust solution the exit code of the `cp` operation could be checked and the loop could continue even if the `cp` step failed.

**Comment by df59** [comment]
Just adding that I ran into this same exact issue. I first opened Oblivion without forcing Proton 6.3, then ran the install script. I then tried removing the mod-organizer-2-oblivion folder, running Oblivion with Proton 6.3 forced, then rerunning the install script. This caused the error.

I had to uninstall Oblivion and remove all Oblivion files, then reinstall and follow the directions, running Oblivion with Proton 6.3 forced before running the install script. The install script then worked.
